client/jetpack-cloud/sections/partner-portal/license-preview/index.tsx

```diff
@@ -65,6 +71,33 @@ export default function LicensePreview( {
 		dispatch( recordTracksEvent( 'calypso_partner_portal_license_list_copy_license_click' ) );
 	}, [ dispatch, translate ] );
 
+	const assign = useCallback( () => {
+		const redirectUrl = addQueryArgs( { key: licenseKey }, '/partner-portal/assign-license' );
```

Comment:
> Looks good! Just one case has a minor bug: If I issue a license without a default payment method I get redirected to add a card which is expected. But, when I input my card and submit, I get a success message but I don't get redirected back and the submit button stays disabled with a "Please wait..." message. ![Screen Shot 2022-06-15 at 18 47 19](https://user-images.githubusercontent.com/22746396/173871457-b76962ed-9ba1-46a7-81dd-07b4aa50f3ff.png)

Sorry, it's been so long that even I forgot how to properly test this and  I haven't even tested right. It should now work fine since I added a fetch for the stored cards after the successful card addition, which then triggers the update of the `has_valid_payment_method` flag and redirects the user back to license issuing.

Redirecting the user to license issuing is not the final goal for the agency signup process, but small adjustments to the user flow will be done on the next updates.